nestkernel/connector_model_impl.h

```diff
@@ -216,9 +224,21 @@ GenericConnectorModel< ConnectionT >::add_connection( Node& src,
   double_t delay,
   double_t weight )
 {
-  if ( not numerics::is_nan( delay ) && has_delay_ )
-    kernel().connection_builder_manager.get_delay_checker().assert_valid_delay_ms( delay );
-
+  if ( not numerics::is_nan( delay ) )
+  {
+    if ( has_delay_ )
+    {
+      kernel().connection_builder_manager.get_delay_checker().assert_valid_delay_ms( delay );
+    }
+    // Let connections without delay contribute to min_delay with wfr_comm_interval
```

Comment:
to min_delay and max_delay, right? However the contribution to max_delay is not important, so I am not sure if we should mention both here!?